README.md

```diff
@@ -366,6 +366,16 @@ Machine.successors
 }
 ```
 
+## Class constants
+Each machine's state will turn into a constant:
+```ruby
+Machine.state(:some_state, initial: true)
+Machine.state(:another_state)
+
+Machine::SOME_STATE #=> "some_state"
+Machine::ANOTHER_STATE # => "another_state"
```

Comment:
Generally think this is a good idea, would be nicer if the states were in their own module so this would become

```ruby
Machine::States::SOME_STATE
```

Reply:
I'm not sure if it would be nicer because these constants are already inside the specialized class, the StateMachine, so it becomes  `PaymentStateMachine::PAID`, and I think it follows the current code organization, we have code like: 
- `PaymentStateMachine.initial_state`
- `PaymentStateMachine.states`
- `PaymentStateMachine.successors` 

And not:
- `PaymentStateMachine::States.initial_state`
- `PaymentStateMachine::States.all`
- `PaymentStateMachine::States.successors`

What do you think? If you are sure that it will better inside the States module, I can make the changes.

Reply:
Thinking about it a little more I think you're right, its probably less unexpected to have the constant in the same module as the call to `state`. I also noticed this is pattern we've unofficially adopted in a few places